README.adoc

```diff
@@ -256,6 +257,15 @@ The following slide deck examples have been exported using DeckTape:
 |https://astefanutti.github.io/decktape/examples/new-es5-overloards.pdf[new-es5-overloards.pdf] (0.1MB)
 |===
 
+=== WebSlides
```

Comment:
Is it intentional you've moved it to the _Examples_ section? I would logically move it within the _Commands_ section and remove it later when DeckTape is based on PhantomJS 2.5 with full Flexbox support. That being said, if you feel like it's too much in the command section, it's fine in the _Examples_ section as it's temporary anyway.